File: 0075. Sort Colors/solution2.java
import java.util.*;

class Solution {
    public void sortColors(int[] nums) { //NOT RECOMMENDED
        List<Integer> l = new ArrayList<>();
        for(int i=0;i<nums.length;i++)
        l.add(nums[i]);

        Collections.sort(l);
        int i=-1;
        for(Object o: l)
        {
            nums[++i] = (int)o;
        }

    }
}

File: 0075. Sort Colors/solution2.cpp
#include<bits/stdc++.h>
using namespace std;

class Solution { //NOT RECOMMENDED
public:
    void sortColors(vector<int>& nums) {
        multiset<int> s;
        for(auto i: nums)
        s.insert(i);
        nums.clear();
        for(auto i: s)
        nums.push_back(i);
    }
};

int main() {

}